src/components/list/List.js

```diff
@@ -1,13 +1,27 @@
 import './list.scss';
 
-const List = ({ items, clickHandler }) => (
-    items ? (
+const List = ({ items, clickHandler, userData, btnHandler }) => (
+
+  items ? (
+    <div className="userInfo">
       <ul className="usersList">
         {
           items.map(el => <li key={el.id} onClick={() => clickHandler(el)}>{el.name}</li>)
         }
       </ul>
-    ) : null
-  );
-  
-  export default List;
\ No newline at end of file
+      {
+        userData.length !== 0
+          ? (
+            <div className={userData.length !== 0 ? 'userPosts' : ''}>
+              <h2>Posts</h2>
+              {userData.length !== 0 ? userData.map(el => <><h4>{el.title}</h4><p>{el.body}</p></>) : null}
```

Comment:
no need to check `userData.length !== 0`
also use `&&` for conditional rendering, no need to use null

Reply:
i decided to check userData.length to not show empty Posts block.